vyper/compiler/output.py

```diff
@@ -108,9 +108,15 @@ def build_integrity(compiler_data: CompilerData) -> str:
 def build_external_interface_output(compiler_data: CompilerData) -> str:
     interface = compiler_data.annotated_vyper_module._metadata["type"].interface
     stem = PurePath(compiler_data.contract_path).stem
+
     # capitalize words separated by '_'
     # ex: test_interface.vy -> TestInterface
-    name = "".join([x.capitalize() for x in stem.split("_")])
+    def capitalize_part(part):
+        if len(part) > 0:  # safe even if 1 cause [1:] will be just empty string
```

Comment:
what about `foo_bar_baz.vy`?

Reply:
it becomes `FooBarBaz`, should it be something different?

Reply:
ah, i see, it's used inside the comprehension